pkg/db/v1beta1/postgres/postgres_test.go

```diff
@@ -129,11 +129,60 @@ func TestDeleteObservationLog(t *testing.T) {
 }
 
 func TestGetDbName(t *testing.T) {
-	// dbName := "root:@tcp(katib-mysql:3306)/katib?timeout=5s"
-	dbName := "host=katib-postgres port=5432 user=katib password= dbname=katib sslmode=disable"
-
-	if getDbName() != dbName {
-		t.Errorf("getDbName returns wrong value %v", getDbName())
+	cases := map[string]struct {
+		updateEnvs map[string]string
+		wantName   string
+	}{
+		"All parameters are default": {
+			updateEnvs: map[string]string{},
```

Comment:
```suggestion
```

I think this initialization isn't needed.

Reply:
Done, remove the initialization.